cmd/ipfs/daemon.go

```diff
@@ -92,7 +92,8 @@ do this by setting headers on the API.HTTPHeaders and Gateway.HTTPHeaders
 keys:
 
 	ipfs config --json API.HTTPHeaders.X-Special-Header '["so special :)"]'
-	ipfs config --json Gateway.HTTPHeaders.X-Special-Header '["so special :)"]'
+	ipfs config --json Gateway.HTTPHeaders.X-Special-Header '["so special
+	:)"]'
 
```

Comment:
It probably not a good idea to wrap here.

I personally would remove the tab and replace it with two (maybe up to four) spaces to help solve the problem to keep it all on a single line.

As an aside I would not use tabs in help text as the width of tab characters is not fixed.

But I may not speak for others.  @whyrusleeping thoughts.